managedapps/spec.go

```diff
@@ -0,0 +1,81 @@
+package managedapps
+
+import (
+	"context"
+
+	"github.com/giantswarm/microerror"
+	"k8s.io/client-go/kubernetes"
+)
+
+type Clients interface {
+	// K8sClient returns a properly configured control plane client for the
+	// Kubernetes API.
+	K8sClient() kubernetes.Interface
+}
+
+// Chart is the chart to test.
+type Chart struct {
+	Name        string
+	URL         string
```

Comment:
e2etests would get full URL only. each e2etest on operators should supply the URL.

Reply:
@rossf7  To list the file in github, I think we need to have github API token. does `e2eseteup` already take into account of that feature?

Reply:
Yes, we already do in this in operator e2e tests to get the release index. So we know the WIP and active version bundles.

e.g. https://github.com/giantswarm/aws-operator/blob/82a58529f7c24b071ef37de26c0681262f86a51d/integration/test/scaling/project.yaml#L11

We can use the same approach.

Reply:
Sure, then I agree to have below features inside `e2esetups`
- Listing app catalog *.tar and find a file which contains same hash.